app/models/trainee.rb

```diff
@@ -198,7 +198,29 @@ class Trainee < ApplicationRecord
   audited associated_with: :provider
   has_associated_audits
 
-  auto_strip_attributes :first_names, :last_name, :postcode, squish: true
+  auto_strip_attributes(
+    :first_names,
+    :middle_names,
+    :last_name,
+    :address_line_one,
+    :address_line_two,
+    :town_city,
+    :postcode,
+    :email,
+    :international_address,
+    :ethnic_background,
+    :additional_ethnic_background,
+    :trn,
+    :additional_withdraw_reason,
+    :region,
+    :hesa_id,
```

Comment:
could consider adding `:additional_disability` too which is an optional text input when adding disability